### PR TITLE
Always pass layout as a string to `setup_test_game`

### PR DESCRIPTION
**Pull request by jakobj**
See https://github.com/ASPP/pelita/pull/515

### CODE REVIEW

**Comment by jakobj** [comment]
oops. done now, pls check

File: README.md [hunk 1]
```diff
@@ -97,106 +97,104 @@ def test_stays_there():
...
-    # # #             #       ## # #
-    #   #   #  #   ## #            #
-    # # ## #####   ## ######       #
-    #                         #    #
-    ################################
-    ```
-    Now you can copy and paste this string in a test, pass it to `setup_test_game` and verify that your bot returns the move you were expecting.
+Given a layout string, returns a [Game](#the-game-object) instance suitable to be passed to a [move](#the-move-function) function. Printing a `Game` object will print its layout string. In the simplest form a layout string is a multiline string where the character `#` identifies walls, `.` the food pellets, `E` the enemy bots (you must have two of them for a layout to be legal), and `0` and `1` representing the bots in your team corresponding to turn `0` and `1`.
+
+For example a maze `8x4` with our bots in `(1, 1)` and `(1, 2)`, where the enemies are on `(5,2)` and `(6,2)` and food pellets in `(2, 2)` and `(6,1)`, and an additional wall in `(4,1)` will look like this:
```
Comment (Debilski) [inline README.md]: What happened to the indentation?

Comment (jakobj) [reply README.md]: i removed it since this is not a list of functions any more. was that not correct?

Comment (Debilski) [reply README.md]: I don’t know. Just wanted to make sure it was not an oversight.

**Pull request merged**